src/api/Controllers/UploadController.cs

```diff
@@ -75,6 +76,9 @@ public async Task<ActionResult<int>> UploadFileAsync(int workgroupId, IFormFile
                         Started = DateTime.Now.ToUniversalTime(),
                         Finished = null,
                     });
+
+                // Set coordinates for missing reference system.
+                await coordinateService.MigrateCoordinatesOfBorehole(borehole, false).ConfigureAwait(false);
```

Comment:
PP: Für boolean Argumente _named Arguments_ verwenden, Hier z.B.
```suggestion
                await coordinateService.MigrateCoordinatesOfBorehole(borehole, onlyMissing: false).ConfigureAwait(false);
```

PP: Ich hätte diesen Aufruf in `UpdateBoreholeLocation` gelassen damit das Setzen von Location und Municipality etc. nicht so verteilt ist.